x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/termsenum/action/MultiShardTermsEnum.java

```diff
@@ -96,21 +101,13 @@ public BytesRef next() throws IOException {
         } else {
             current = null;
         }
-
         return current;
     }
 
-    public long docFreq() throws IOException {
```

Comment:
Need to adjust the javadoc at the top of the class to not refer to this anymore?